frontend/.eslintrc.json

```diff
@@ -44,6 +44,7 @@
       "import/no-extraneous-dependencies": ["error", {"devDependencies": true}],
       "max-len": ["error", {"comments": 100, "code": 130}],
       "import/no-unresolved": ["error",{"ignore": [".scss"]}],
-      "react/jsx-props-no-spreading": ["off"]
+      "react/jsx-props-no-spreading": ["off"],
+      "react/jsx-wrap-multilines": ["error", {"prop": "ignore"}]
```

Comment:
I added this so things like the below are acceptable:
```jsx
<Component
  label={
    <div>
      <span>Ho ho ho</span>
    </div>
  }
/>
```

Without this, the linter complains about about missing parens but, when you add them, a different rule complains that they are unnecessary.